spec/internal/db.lua

```diff
@@ -277,6 +277,8 @@ end
 -- }
 local function get_db_utils(strategy, tables, plugins, vaults, skip_migrations)
   strategy = strategy or conf.database
+  conf.database = strategy
```

Comment:
Could you add a comment for this change? It may be helpful for the future maintainers.